common/src/main/java/org/apache/atlas/repository/Constants.java

```diff
@@ -324,6 +324,8 @@ public enum SupportedFileExtensions { XLSX, XLS, CSV }
 
     public static final int ELASTICSEARCH_PAGINATION_SIZE = 50;
 
+    public static final String DELETE_HANDLER_TYPE_HARD = "HARD";
```

Comment:
This also can be replaced with Enum instead of string constant
Use DeleteType.HARD.name() wherever referred this string constant